tests/testthat/test_hector.R

```diff
@@ -0,0 +1,70 @@
+context("Expansion of test code to include all files")
```

Comment:
This isn't really a context, or at least not a useful one. If we need to break this file into multiple files so that there's a single coherent context, let's do that please.

Reply:
Or change context to "Miscellaneous" if that's the best course?